examples/examples_mfa_hierarchy/in/nnig_ngg.asciipb

```diff
@@ -0,0 +1,13 @@
+fixed_values {
+  mutilde: {
+    size:10
+    data:[1.0, 1.0, 1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0]

```

Comment:
I'm not sure this is the right way to specify a repeated field, or is it? I know for sure that having

```
data: 1.0
data: 1.0
data: 1.0
...
```
and so on (for 10 times) would work

Reply:
I looked around a bit on the Internet and I did find some file which uses this notation, so I guess it's fine!